src/databricks/labs/ucx/assessment/clusters.py

```diff
@@ -34,6 +34,8 @@ class ClusterInfo:
     cluster_id: str
     success: int
     failures: str
+    spark_version: str
+    policy_id: str | None = None
```

Comment:
adding new fields is a breaking change. add the upgrade script for it: see https://github.com/databrickslabs/blueprint?tab=readme-ov-file#application-state-migrations